examples/basic-axios/pages/api/data.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
+
+const projects = [
+  'facebook/flipper', 'vuejs/vuepress', 'rust-lang/rust', 'zeit/next.js'
+]
+
+export default (req, res) => {
```

Comment:
From my understanding of the code the fetcher-function must return an awaitable. This returns `undefined` and will not work.

Reply:
> fetcher-function must return an awaitable

Not really.
Note, that this is nodejs server handler, and it doesn't matter what is returned - the key part is `res.json` call, which ends the request

Reply:
My bad, I looked at the wrong file. Sorry.

Reply:
No worries :)